### DISCUSSION TITLE
Vibration sensor drive

### DISCUSSION THREAD

**Discussion by 1486756632** [edgexfoundry/community#2] [Q&A]
i have a project.i need to collect data from a Vibration sensor.Is there anyone got a Vibration sensor drive.

**Comment by farshidtz** [comment]
What vibration sensor do you intend to use? It may help if you provide more details on the devices involved in your use case.

For a sensor connected to a micro controller, you can write a simple program to collect that data and pass it to an MQTT broker. Then you can use EdgeX's [MQTT device service](https://github.com/edgexfoundry/device-mqtt-go) to ingest the data into EdgeX.
